src/plone/importexport/tests/test_robot.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: UTF-8 -*-
-from plone.importexport.testing import PLONE_IMPORTEXPORT_ACCEPTANCE_TESTING  # noqa
 from plone.app.testing import ROBOT_TEST_LEVEL
+from plone.importexport.testing import PLONE_IMPORTEXPORT_ACCEPTANCE_TESTING  # noqa
```

Comment:
could you please mask all E501 Errors that should stay because of semantics to 

    # NOQA: E501

Reply:
Okay, working on it. @loechel